app/views/katello/api/v2/module_streams/compare.json.rabl

```diff
@@ -0,0 +1,10 @@
+object false
+
+extends "katello/api/v2/common/metadata"
+
+child @collection[:results] => :results do
+  extends 'katello/api/v2/module_streams/base'
+  node :comparison do |result|
```

Comment:
@jeremylenz as for the counts, I initially had the counts for each content type tab. However, after looking at the mocks, noticed that there are no counts there. So, I removed them. Here is the link to the mocks, if you need it: https://marvelapp.com/prototype/6249jcd/screen/87503017

Reply:
Makes sense about the counts. I see that now on the mockups. thanks!

Reply:
@jeremylenz @sjha4 would like to follow up on the comment about renaming " from comparison to appears_in". Should I make these changes?

Reply:
I am not strongly for or against either comparison or appears_in. My only concern would be changing it right away in case someone somewhere is using the API today. I don't mind adding a deprecation warning and changing it in the next release after some more feedback.

Reply:
@sjha4 Sounds good. I will push my code with most of the things addressed in this PR except for this comment.

Reply:
Opened https://projects.theforeman.org/issues/35314 to track the rabl node change.